src/cpp/redis.cpp

```diff
@@ -106,8 +106,7 @@ bool Redis::key_exists(const std::string& key)
 {
     // Build the command
     SingleKeyCommand cmd;
-    cmd.add_field("EXISTS");
-    cmd.add_field(key);
+    cmd << "EXISTS" << key;
```

Comment:
Should this be `cmd << "EXISTS" << KeyField(key);`?

Reply:
@billschereriii ?

Reply:
This is murky. The original code did NOT mark this key as a key, and I'm not clear on why. I need to check in with MattE on why to decide whether it should be marked as a key here.

Reply:
We can add this as a ``Keyfield`` if we want.  The ``Redis`` class doesn't need it to be accurate because all requests go to a single node.  I don't think there is a performance impact either way.